test/ui/ViewJobRolesPage.ts

```diff
@@ -0,0 +1,65 @@
+import { By } from "selenium-webdriver";
+import BasePage from "./BasePage";
+
+interface JobRole {
+  roleName: string;
+  location: string;
+  band: string;
+  capability: string;
+  closingDate: string;
+}
+
+export class JobRolesPage extends BasePage {
+  //private url: string = 'https://5chmbvngab.eu-west-1.awsapprunner.com/job-roles';
+  private url: string = "http://localhost:3000/job-roles";
```

Comment:
You could have something like this: `private url: string = process.env.UI_TEST_URL || '<aws_url>/job-roles';`